srcs/response/Response.cpp

```diff
@@ -132,13 +133,38 @@ OutgoingData * Response::handleGet(const t_server & server, const HTTPRequest &
 OutgoingData * Response::handlePost(const t_server & server, const HTTPRequest & req)
 {
 	(void)server;
-	(void)req;
-	return NULL;
+	std::string uri = req.getUriWithoutQString();
+	const t_route *routeptr = getRouteFromUri(server, uri);
+	if (routeptr == NULL) {
```

Comment:
Please be consistent with brackets, either haskell or something else

Reply:
bruuhhh